spec/api/v2/screening_lists/consolidated_spec.rb

```diff
@@ -308,5 +308,33 @@
       end
     end
 
+    context 'when expiration_date is specified' do
+      subject { response }
+      let(:params) { { sources: 'SDN', expiration_date: '[* TO 2011-01-23]' } }
```

Comment:
BTW, a suitable solution may be to:

- keep the ES-like range syntax in the query string.
- break this into its component start and end parts in the query (raising an error if the format is invalid), then turning into a filter->range query for ES.

That way, you solve both issues I've raised about this (turning filter into query is bad for performance, not validated input from outside is dangerous).